mainapp/models.py

```diff
@@ -184,3 +182,43 @@ class DistrictCollection(models.Model):
     collection = models.TextField(
         verbose_name="Details of collected items"
     )
+
+class RescueCamp(models.Model):
+    verbose_name = 'Relief Camp'
+    name = models.CharField(max_length=50,verbose_name="Name")
+    location = models.TextField(verbose_name="Address",blank=True,null=True)
+    district = models.CharField(
+        max_length=15,
+        choices=districts
+    )
+    contacts = models.TextField(verbose_name="Phone Numbers",blank=True,null=True)
```

Comment:
I'd suggest using a Phone model and do ManyToManyField there rather than a `Textfield` - Would be much easier to format/show/handle later.
```python
class PhoneNumber(models.Model):
    phone = models.CharField(blank=True, max_length=11, null=True, verbose_name='Mobile - മൊബൈൽ'))
```